web/templates/events/event.html

```diff
@@ -30,10 +30,14 @@
             <p><a href="http://koodikerho.fi/">http://koodikerho.fi/</a></p>
           {% endif %}
           <dl class="dl-horizontal event-description-list">
+            <dt>Ilmoittautuminen aukeaa:</dt>
+            <dd>{% load tz %}{% localtime on %}{{ event.signup_open }}{% endlocaltime %}</dd>
```

Comment:
This could be checked that does it exist. So `{% if event.signup_open %}`

Otherwise LGTM :+1: